tests/python/test_bit_struct.py

```diff
@@ -36,7 +36,7 @@ def verify_val():
     verify_val.__wrapped__()
 
 
-@ti.test(require=ti.extension.quant_basic, debug=True)
+@ti.test(require=ti.extension.quant_basic, exclude=[ti.metal], debug=True)
```

Comment:
Nit: if we exclude a backend, it'd be nice if we can add a note whether we need to fix this later (like if it's supposed to be excluded) ;)

Reply:
I'll start a new issue